iis-wf/iis-wf-transformers/src/main/resources/eu/dnetlib/iis/wf/transformers/common/union/oozie_app/workflow.xml

```diff
@@ -13,6 +13,12 @@
 		<property>
 			<name>schema</name>
 		</property>
+		<property>
+			<name>combine_splits</name>
+			<!-- zero will use the Pig default -->
+			<value>0</value>
```

Comment:
0 is neat. I was afraid we would need to provide current default value explicitly.